test/support/api/states.js

```diff
@@ -20,6 +20,7 @@ export const stateTodo = _.find(states, { id: '22222' });
 export const stateInProgress = _.find(states, { id: '33333' });
 export const stateDone = _.find(states, { id: '55555' });
 export const stateUnableToComplete = _.find(states, { id: '66666' });
+export const stateThmgTransfered = _.find(states, { id: '77777' });
```

Comment:
I wonder if this whole state could be removed.    Not for this PR, but I think it might be done

Reply:
I made a new story for using uuids for the state & roles fixtures [here](https://app.shortcut.com/roundingwell/story/61215/in-tests-have-roles-states-fixtures-use-uuid-ids). And I included a note to look into removing this.

There's also a `Evernorth` done state with `id: '88888'` that could potentially be removed.